functional-tests/testenv.py

```diff
@@ -45,6 +58,161 @@ def __init__(self, datadir_root: str, name: str, env: flexitest.LiveEnv):
         super().__init__(env)
 
 
+class BridgeTestBase(StrataTester):
+    """
+    Testbase for bridge specific test.
+    Provides methods for setting up service, making DRT, withdraw transaction
+    """
+
+    def premain(self, ctx: flexitest.RunContext):
```

Comment:
I can't find any usages for this method. Is there some place where this is being called implicitly?

Reply:
Yes, `premain(ctx)` is part of the flexitest `TestRuntime`.
https://codeberg.org/treyd/flexitest/src/commit/216e87270994ebb023ba9b0c1a38134cc59da1f7/flexitest/runtime.py#L164

Reply:
TIL

Reply:
Yeah this is somewhat new.